models/service_model.go

```diff
@@ -35,6 +35,7 @@ const (
 	PostgreSQLServiceType ServiceType = "postgresql"
 	ProxySQLServiceType   ServiceType = "proxysql"
 	ExternalServiceType   ServiceType = "external"
+	HAProxyServiceType    ServiceType = "haproxy"
```

Comment:
let's set haproxy above external here and in other places.

Reply:
Moved up to external.